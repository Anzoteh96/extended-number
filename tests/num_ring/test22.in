-999990081673799860096244489874953368154761580300221998041596954914
-336823838267584955834026039377734934942187918514163327005296074097
190572939056142734383890932788526598447209565136129136610726115287
