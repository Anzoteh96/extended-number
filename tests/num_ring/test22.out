Enter the number base b:
Enter two numbers n and m:
b is 999990081673799860096244489874953368154761580300221998041596954914
n mod b is 663166243406214904262218450497218433212573661786058671036300880817
m mod b is 190572939056142734383890932788526598447209565136129136610726115287
n + m is 853739182462357638646109383285745031659783226922187807647026996104
n - m is 472593304350072169878327517708691834765364096649929534425574765530
n * m is 327897151231411295283341575686639493224044887088938827949332982383
n ** m is 560122397655116085867757259186252539945383225817753552886675478125
